src/contracts/core/AllocationManager.sol

```diff
@@ -313,101 +285,49 @@ contract AllocationManager is
             );
 
             bytes32 operatorSetKey = _encodeOperatorSet(allocation.operatorSets[i]);
-            // prep magnitudeUpdates in storage
-
-            // Check that there is at MOST `MAX_PENDING_UPDATES` combined allocations & deallocations for the operator, operatorSet, strategy
-            (uint256 numPendingAllocations, uint256 numPendingDeallocations) =
-                _getPendingAllocationsAndDeallocations(operator, allocation.strategy, operatorSetKey);
 
+            // Check that there are no pending allocations & deallocations for the operator, operatorSet, strategy
+            MagnitudeInfo memory opsetMagnitudeInfo = _operatorMagnitudeInfo[operator][allocation.strategy][operatorSetKey];
             require(
-                numPendingAllocations + numPendingDeallocations < MAX_PENDING_UPDATES,
+                opsetMagnitudeInfo.effectTimestamp <= block.timestamp,
                 PendingAllocationOrDeallocation()
```

Comment:
Maybe we could make this error a little more clear -- `ModificationAlreadyPending` or something?

Reply:
Fixed.